recipes/mmcif-pdbx/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "mmcif_pdbx" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a2f2abb8e62fe7b78acd9114be74f4ade2f29e971079b3e817f53b9b9b21eacc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - pdbx
+
+about:
+  home: "https://github.com/Electrostatics/mmcif_pdbx"
+  license: CC0
+  license_family: PUBLIC-DOMAIN
+  license_file: LICENSE
+  summary: "Python utilities for PDBx/mmCIF storage model"
+  doc_url: "http://mmcif.wwpdb.org/pdbx-mmcif-home-page.html"
+  dev_url: "https://github.com/Electrostatics/mmcif_pdbx"
+
+extra:
+  recipe-maintainers:
+    - sobolevnrm
```

Comment:
@sobolevnrm are you OK with being a maintainer?

Reply:
Yes!  However, I very much appreciate help from @tsenapathi in getting this going and will likely reply on his support in the future (if he's willing).  Thank you.

Reply:
@sobolevnrm and @scopatz  Yes I am willing for my continue my support in future (I have added myself to the maintainers in here).